examples/vl3_basic/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -46,14 +50,12 @@ spec:
               value: "nsmgr.nsm-system"
             - name: NSREGISTRY_PORT
               value: "5000"
+            - name: IPAM_ADDRESS
+              value: "ipam-{{ .Values.cnns.nsr.addr }}:50051"
```

Comment:
don't tie this to the NSR in the helm chart.  The scripts that use the chart might be able to do that but not the chart.

Reply:
The IPAM server address should go in the `ucnf-vl3-{{ .Values.nsm.serviceName }}` configmap below under the ipam section should be a `serverAddress` value.  See github.com/danielvladco/k8s-vnet@v0.3.0/pkg/nseconfig/config.go L40 `type IPAM`